CorinneTest/Factory.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace CorinneTest
+{
+    class Factory : IPersonFactory
+    {
+       
+        static Random rnd = new Random();
+
+
+        public static List<Person> CreateInRange(int lowAge,
+                                                 int highAge,
+                                                 int? numOfPeople)
+        {
+            var list = new List<Person>();
+            int i = 0;
+            while (i < numOfPeople)
```

Comment:
where do you increment `i` to allow breaking out of the loop?

Reply:
oops - changed this to increment i outside of the switch statement.